src/reducers/index.js

```diff
@@ -25,33 +23,16 @@ import { DASHBOARD_CHART_TIME } from '../constants';
 */
 
 const initialState = {
-  data: [],
+  data: null,
   isVersionAllowed: undefined,
 };
 
 const rootReducer = (state = initialState, action) => {
   switch (action.type) {
     case 'dashboard_update':
-      let data = state.data.slice(0);
       let newData = action.payload;
       newData['date'] = new Date(newData.time*1000);
```

Comment:
I think we don't need this anymore, as it was used for the charts.

Reply:
That's true, done.